src/interpreter_funcs/voxel_boolean_difference.rs

```diff
@@ -154,7 +178,48 @@ impl Func for FuncBooleanDifference {
         let voxel_dimensions = args[2].unwrap_float3();
         let growth_iterations = args[3].unwrap_uint();
         let fill = args[4].unwrap_boolean();
-        let analyze = args[5].unwrap_boolean();
+        let warn_if_slow = args[5].unwrap_boolean();
+        let analyze_bbox = args[6].unwrap_boolean();
+        let analyze_mesh = args[7].unwrap_boolean();
+
+        let bbox_diagonal1 = mesh1.bounding_box().diagonal();
+        let voxel_count1 = (bbox_diagonal1.x / voxel_dimensions[0]).ceil() as u32
+            * (bbox_diagonal1.y / voxel_dimensions[1]).ceil() as u32
+            * (bbox_diagonal1.z / voxel_dimensions[2]).ceil() as u32;
+
+        let bbox_diagonal2 = mesh2.bounding_box().diagonal();
+        let voxel_count2 = (bbox_diagonal2.x / voxel_dimensions[0]).ceil() as u32
+            * (bbox_diagonal2.y / voxel_dimensions[1]).ceil() as u32
+            * (bbox_diagonal2.z / voxel_dimensions[2]).ceil() as u32;
+
+        let (voxel_count, bbox_diagonal) = if voxel_count1 > voxel_count2 {
+            (voxel_count1, bbox_diagonal1)
+        } else {
+            (voxel_count2, bbox_diagonal2)
+        };
+
+        log(LogMessage::info(format!("Voxel count = {}", voxel_count)));
+
+        if warn_if_slow && voxel_count > VOXEL_COUNT_THRESHOLD {
+            let vy_to_vx = voxel_dimensions[1] / voxel_dimensions[0];
```

Comment:
Correct me if I misunderstand, but if this is supposed to be a transformation, then it transforms from x to y, hence the name should be `vx_to_vy`. Same everywhere this was introduced.

Reply:
it's `y / x`, so ti's an *y to x ratio*. not sure how to call it correctly

Reply:
Haha, I have lived with the mstrices for too long. Ok, ignore this.

Reply:
renamed to `vy_over_vx`